packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -97,3 +154,4 @@ ShowHide.propTypes = propTypes;
 ShowHide.defaultProps = defaultProps;
 
 export default injectIntl(ShowHide);
+export { ShowHideFocuser };
```

Comment:
Why is the ShowHide component exporting the ShowHideFocuser component as well?

Reply:
The ShowHideFocuser provides additional HTML structure around text to allow JAWS walk throw the text with Down Arrow. The user needs to import it along with ShowHide component from the show-hide package. Exporting ShowHideFocuser from the ShowHide.jsx allows to import it as one line with ShowHide like this:

`import ShowHide, {ShowHideFocuser} from 'terra-show-hide';`
similarly to how we import this:
`import React, {useState} from 'react'; `

Reply:
@sdadn index file added: https://github.com/cerner/terra-core/pull/3838/commits/822fe0339ce00a4d80c2b649f25b3dc2def71971